azure-pipelines/build-release.yml

```diff
@@ -129,9 +129,8 @@ stages:
             inputs:
               versionSpec: '3.7'
 
-          - template: steps/install-development-dependencies.yml
-
           - script: |
+              python -m pip install --upgrade tox
```

Comment:
Commit "ci: Use tox instead of pipenv for checknews" doesn't remove pipenv. We should rename the commit to "ci: Use tox to run the news check"